pkg/download/classic/downloader.go

```diff
@@ -49,83 +49,37 @@ type Downloader struct {
 	client dtclient.Client
 }
 
-// WithAPIContentFilters sets the api content filters for the Downloader - these will be used in addition to base API filtering
-func WithAPIContentFilters(apiFilters map[string]contentFilter) func(*Downloader) {
```

Comment:
Should this option really be removed?
Seemingly they're only used in tests right now as production code defaults, but on an interface level these options make sense. I'd keep it, rather than moving it into a non-exported test only method

Reply:
As you said, for now, the only purpose is in testing. For that reason I moved it to the test file.

It is moved back!